src/containers/Trigger/Trigger.tsx

```diff
@@ -23,6 +23,19 @@ import { CREATE_TRIGGER, DELETE_TRIGGER, UPDATE_TRIGGER } from '../../graphql/mu
 export interface TriggerProps {
   match: any;
 }
+const checkDateTimeValidation = (startAtValue: any, startDateValue: any) => {
```

Comment:
```suggestion
const checkDateTimeValidation = (startAtValue: string, startDateValue: string) => {
```
Let's try to use types wherever we see scope🙂